code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,24 +92,50 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+	var/ref_client = "\ref[src]"
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "No")
+			if(alert(usr,"You already have an adminhelp open, would you like to bump it?", "Bump Adminhelp", "Yes", "No") == "Yes")
+				T.logs += "[src.ckey] has bumped this adminhelp!"
+				if(T.admin == "N/A")

```

Comment:
These are actually defined under datum/adminticket.  and as far as I can tell are use for ensuring not more than one admin takes over a ticket at a time.